### ISSUE TITLE
Problem installing Certbot on Jitsi-meet on Ubuntu 17.10 - won't verify domains

### ISSUE THREAD

**Issue by leehclark3** [jitsi/jitsi-meet#2573]
I installed fresh Ubuntu 17.10 on laptop, then Jitsi-meet. Then when I try to install Certbot it won't verify domain. I have no web-server installed. I'm not proficient with commands but I can copy and paste. Should I install a web-server? Do I need to create folders and give the right permissions?I just have computers at home that I use with a static IP. I just use A records with the static IP for my DNS configuration. Can someone tell me how to serve secure Jitsi-meet video from my little home setup? I am flexible. Just need someone to walk me through this. I've seen other similar issues addressed here but the explanations are over my head. Just tell me what I need to do. You can't assume I know what I'm doing because I have only the most rudimentary knowledge and no formal training of any kind.

I ran /usr/share/jitsi-meet/install-letsencrypt-cert.sh as root and I still cannot get a domain verification. unknown host error. Also tried the Tutorial for Building Out an install from scratch: Ubuntu 

Obtaining a new certificate
Performing the following challenges:
http-01 challenge for recoverylive.webcam
Waiting for verification...
Cleaning up challenges
Failed authorization procedure. recoverylive.webcam (http-01): urn:acme:error:unknownHost :: The server could not resolve a domain name :: No valid IP addresses found for recoverylive.webcam

IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: recoverylive.webcam
   Type:   unknownHost
   Detail: No valid IP addresses found for recoverylive.webcam

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A/AAAA record(s) for that domain
   contain(s) the right IP address.
 - Your account credentials have been saved in your Certbot
   configuration directory at /etc/letsencrypt. You should make a
   secure backup of this folder now. This configuration directory will
   also contain certificates and private keys obtained by Certbot so
   making regular backups of this folder is ideal.16.04/Apache2 #2531, same problem.
--

**Comment by damencho** [comment]
Apparently your Dns is not setup correctly. You need to fix your A record. 
I made a quick query for your domain using online dns reporting tool and I see only mx records for your domain, no A records.

**Comment by leehclark3** [comment]
i have A records for www & @ pointing to my IP

**Comment by leehclark3** [comment]
just started server

**Comment by leehclark3** [comment]
why do you keep closing my issues? this is the second time and both times
your answers have not been valid. i tried that script and my dns is
correct.

**Comment by damencho** [comment]
Now I see them. 
You also need a public address in your A record. Let's encrypt need to be able to connect to your server, to verify that the server you claim is really yours.

**Comment by leehclark3** [comment]
can you tell me how to do that? what's a public address? i'm using my
static ip. i suspect the issue is about having the permissions set up in
ubuntu. you can get to mt jitsi meet room but it's not secure.

**Comment by damencho** [comment]
Go to https://www.whatismyip.com/ is the address you see there the same as the one in your A record?
Go to http://www.dnsstuff.com/tools#dnsReport|type=domain&&value=recoverylive.webcam
And see WWW section:

FAIL | IP Address(es) valid | All addresses are private. That means that all people trying to get to your web site will not be able to get there. The records are:172.16.0.10
-- | -- | --

The first time I tested I was on mobile and used wrong dns tool, that's why I saw only MX records. Everybody can make mistakes. 

I'm closing issues, cause I'm sure what is the problem, I clearly see the error that Letsencrypt server cannot access your machine. And without closing issues, we end up with this unmanaged number of issues '366' that I see now, cause people keep opening issues for all sorts of things and do not close them. 
As this is not an issue with the code, this is a support ticket, a problem with configuration, it is even not a configuration for our product, but I continue giving support, that's why I closed it.
And I continue answering those closed issues when someone asks a question there as you currently see.
